ethereum/README.md

```diff
@@ -0,0 +1,22 @@
+This repository has the contracts that implement the wrapped btc token.
```

Comment:
Let's try to include that this is the original version and the deployed version.